Modules/Core/Libraries/Sumex.php

```
@@ -291,9 +292,10 @@ public function pdf($invoice_template = null): bool|string
             'AFRelationship' => 'Alternative',
         ]];
 
+        $invoiceTaxRateService = app(\Modules\Invoices\Services\InvoiceTaxRateService::class);
```

PHPMD (2.15.0)

Avoid excessively long variable names like $invoiceTaxRateService. Keep variable name length under 20. (undefined)

(LongVariable)